webrtc.html

```diff
@@ -1807,8 +1807,8 @@
                         </ol>
                       </li>
                       <li>
-                        <p>If <code><var>candidate</var>.ufrag</code> is
-                        non-null, and is not equal to any ufrag present in the
+                        <p>If <code><var>candidate</var>.ufrag</code> is not
+                        missing, and is not equal to any ufrag present in the
```

Comment:
`ufrag` is not nullable so it can never be `null`.

I think we should be explicit, and say "If candidate.ufrag is not `undefined`,"